setup.sh

```diff
@@ -208,6 +208,17 @@ jumpbox() {
         ssh -At -oStrictHostKeyChecking=no ec2-user@$JUMPBOX ssh -oStrictHostKeyChecking=no ec2-user@$INSTANCE_IP
 }
 
+create_console() {
+        echo $1
+        cd $TERRAFORMPROJ$1
+
+        aws-vault exec ${PROFILE_NAME} -- $TERRAFORMPATH console -var-file=$TERRAFORMTFVARS
+
+        if [ $? != 0 ]; then
```

Comment:
I'm not that good at bash but do we need this if statement? If the console statement exits, regardless of the exit code we would exit the function anyway as it's the end of it?

Reply:
The idea behind this peace of code was to enable the ability t catch unforseeen cases. It is not majorly required but since the console is highly dynamic enviroment we can always forsee possible failure cases.

I am happy to remove if you are happy to continue with that risk. This peace of code takes the exit code from the previously run command and checks it. The main purpose if to prevent the code from hanging.